src/pointer-analysis/value_set.h

```diff
@@ -242,28 +242,28 @@ class value_sett
     const namespacet &ns) const;
 
 protected:
-  void get_value_set_rec(
+  virtual void get_value_set_rec(
```

Comment:
Again, I find it extremely unlikely that these methods are already suitable customization points. It might be better to find a more focused group of custom behaviors which can be given private virtual interfaces, and called from these (still non-virtual) methods.